content/collections/audit-works.html

```diff
@@ -1,3 +1,76 @@
+<item>
+  <field name="logo">
+    double.jpeg
+  </field>
+  <field name="title">
+    Double-A-1
+  </field>
+<field name="subtitle">
+  A DeFi primitive for AMM liquidity providing
+  </field>
+  <field name="date">
+    2023-01-20
+  </field>
+  <field name="about">
```

Comment:
Audited Double core vaults, reward distribution, and UniswapV2 liquidity provider and migratory contracts